### ISSUE TITLE
Data Collection

### ISSUE THREAD

**Issue by Folly135** [LanqingLi1993/FOCAL-latest#1]
@LanqingLi1993 Hi! I noticed that you changed the way you build the offline dataset, does the offline dataset in the form of zip files provided in the original FOCAL still work? Do I need to regenerate it?

**Comment by Lagrant** [comment]
Yes, you need to follow the instructions in readme file to regenerate it.
